default.config.yml

```diff
@@ -98,37 +98,36 @@ drupalvm_cron_jobs: []
 # this variable is 'true'.
 configure_drush_aliases: true
 
+apache_vhosts_extra_parameters: |
+    <FilesMatch \.php$>
+      SetHandler "proxy:fcgi://{{ php_fpm_listen }}"
+    </FilesMatch>
```

Comment:
I think that might be helpful... It's so verbose the way it's set up right now :/